Dispatcher/Controller/HeaderViewController.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class HeaderViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let headerHeight: CGFloat = 100
+        let customHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: view.frame.size.width, height: headerHeight))
```

Comment:
working with frames is discouraged 
for now we will build the components with the regular storyboards and xibs